### PR TITLE
citation button

### LINKED ISSUE #1812: Add info button on CollectionCitation
Screen Shot 2021-05-05 at 10 50 27 AM

Followup from PR #1801.

### PR DESCRIPTION
**Pull request by mccalluc**
Does this look ok? Fix #1812 
Screen Shot 2021-05-11 at 3 17 08 PM

### CODE REVIEW

**Comment by tsliaw** [comment]
The info icon looks a little off-center (too high). Do you think you can make it more in-line with the text? Otherwise, PR looks good to go.

**Review by tsliaw — APPROVED** [review]

File: context/app/static/js/components/Detail/Citation/style.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
 
+import { InfoIcon } from 'js/shared-styles/icons';
+
 const FlexPaper = styled(Paper)`
   padding: 30px 40px;
 `;
 
-export { FlexPaper };
+const StyledInfoIcon = styled(InfoIcon)`
+  margin-right: ${(props) => props.theme.spacing(0.5)}px;
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/Citation/style.js:11]: Should this be margin left?

File: context/app/static/js/components/Detail/Citation/Citation.jsx [hunk 2]
```diff
@@ -20,7 +21,13 @@ function Citation({ contributors, title, timestamp, url }) {
 
   return (
     <SectionContainer id="citation">
-      <SectionHeader>Citation</SectionHeader>
+      <SectionHeader>
```
Comment (john-conroy) [inline context/app/static/js/components/Detail/Citation/Citation.jsx:24]: For a quick fix, adding `display: flex; align-items:center;` to a styled section header should fix the alignment issue.

**Review by john-conroy — COMMENTED** [review]
A few thoughts. Let me know if jumping in here is the wrong thing to do.

File: context/app/static/js/components/Detail/Citation/style.js (hunk 1 above)
Comment (mccalluc) [reply context/app/static/js/components/Detail/Citation/style.js:11]: yes... but it looks fine without any style at all. Let me take that out...

File: context/app/static/js/components/Detail/Citation/Citation.jsx (hunk 2 above)
Comment (mccalluc) [reply context/app/static/js/components/Detail/Citation/Citation.jsx:24]: Thanks!

**Pull request merged**